packages/block-library/src/navigation/edit.js

```diff
@@ -171,6 +215,31 @@ function Navigation( {
 							{ __( 'Create from all top-level pages' ) }
 						</Button>
 
+						{ !! hasMenus && (
+							<>
+								<SelectControl
+									label={ __( 'Create from existing Menu' ) }
+									value={ selectedMenu }
+									onChange={ ( value ) => {
+										setSelectedMenu( value );
+									} }
+									options={ menus.map( ( mappedMenu ) => {
```

Comment:
We're going to need to test for the presence of menu items before we output the Menu. Otherwise the `menuItems` is an empty array which doesn't allow us to create any Nav Item Blocks.

Or we simply create an empty Nav

Reply:
I opted to show the Menu but if it's empty simply create an empty Nav